cflib/positioning/position_hl_commander.py

```diff
@@ -117,7 +121,7 @@ def take_off(self, height=DEFAULT, velocity=DEFAULT):
         time.sleep(duration_s)
         self._z = height
 
-    def land(self, velocity=DEFAULT):
+    def land(self, velocity=DEFAULT, landing_height=0.0):
```

Comment:
I think the user would expect a call to `land()` to use the `default_landing_height` that was set when the class was instantiated, the current implementation would try to land at 0.0. To get the desired functionality you will have to follow the same pattern that is used for the `velocity` parameter and set it to `DEFAULT`.